src/compile/selection/transforms/clear.ts

```diff
@@ -0,0 +1,19 @@
+import {TUPLE} from '..';
+import {TransformCompiler} from './transforms';
+
+const clear: TransformCompiler = {
+  has: selCmpt => {
+    return selCmpt.clear;
+  },
+
+  signals: (model, selCmpt, signals) => {
+    const tupleIdx = signals.findIndex(i => i.name === selCmpt.name + TUPLE);
+    signals[tupleIdx].on.push({
+      events: [{source: 'scope', type: selCmpt.clear}],
```

Comment:
Wrap `selCmpt.clear` with a call to `parseSelector` (like in `translate.ts` or `zoom.ts`). This is important to making sure that, in multi-view visualizations, clearing only fires if the user double-clicks within the same view as the selection was defined in (`scope`).